lib/src/binding_mqtt/mqtt_extensions.dart

```diff
@@ -0,0 +1,157 @@
+// Copyright 2022 The NAMIB Project Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import 'package:curie/curie.dart';
+import 'package:mqtt_client/mqtt_client.dart';
+import 'package:mqtt_client/mqtt_server_client.dart';
+import 'package:uuid/uuid.dart';
+
+import '../../core.dart';
+import '../definitions/form.dart';
+import '../definitions/security/auto_security_scheme.dart';
+import '../definitions/security/basic_security_scheme.dart';
+import 'constants.dart';
+import 'mqtt_binding_exception.dart';
+
+/// [PrefixMapping] for expanding MQTT Vocabulary terms from compact IRIs.
+final mqttPrefixMapping = PrefixMapping(defaultPrefixValue: mqttContextUri);
+
+/// Additional methods for making MQTT [Uri]s easier to work with.
+extension MqttUriExtension on Uri {
+  // TODO: Is this the right approach to creating an identifier here?
+  String get _id => const Uuid().v4();
+
+  /// Create an [MqttServerClient] from this [Uri].
+  MqttServerClient createClient() {
+    final client = MqttServerClient.withPort(
+      _brokerAddress,
+      _id,
+      _mqttPort,
+    );
+
+    if (scheme == mqttSecureUriScheme) {
+      client.secure = true;
+    }
+
+    client
+      ..setProtocolV311()
+      ..keepAlivePeriod = 20;
```

Comment:
20 seems like a magic number here

Reply:
Should be addressed in 951378069c335d9f7491e82b1a4d97d2d71b0d54

Reply:
Looks good, the new field `keepAlivePeriod` should be documented, though.

Reply:
Oh, sorry, I missed that 😅 Should be resolved by 2735ca15bffbca251da934a8f76d89f780877ee6 :)